projects/js-packages/shared-extension-utils/package.json

```diff
@@ -16,9 +16,12 @@
 		"test": "NODE_ENV=test NODE_PATH=tests:. js-test-runner --jsdom --initfile=test-main.jsx 'glob:./!(node_modules)/**/test/*.@(jsx|js)'"
 	},
 	"dependencies": {
+		"@wordpress/compose": "5.1.1",
```

Comment:
Let's downgrade this to `5.1.0`, to keep in sync with the other packages in the monorepo that use `@wordpress/compose`